### PR TITLE
JW-952 cleanup

### PR DESCRIPTION
**Pull request by michal-clapinski**

### CODE REVIEW

File: include/vr_common.h [hunk 1]
```diff
@@ -6,7 +6,7 @@
 #include <basetsd.h>
 #include <errno.h>
 
-#ifdef _NTKERNEL
+#ifdef _KERNEL_MODE
```
Comment (jablonskim) [inline include/vr_common.h]: Maybe change it to `__KERNEL__`

File: include/vr_common.h [hunk 2]
```diff
@@ -21,19 +21,21 @@ typedef BOOLEAN bool;
...
 #include <stdint.h>
 #include <stdbool.h>
 
-#endif /* _NTKERNEL */
-
 #define __LITTLE_ENDIAN 1
 #define __BIG_ENDIAN 2
 #define __BYTE_ORDER __LITTLE_ENDIAN
 
+#endif /* _KERNEL_MODE */
```
Comment (jablonskim) [inline include/vr_common.h]: Maybe change it to `__KERNEL__`

File: include/vr_interface.h [hunk 3]
```diff
@@ -250,11 +250,11 @@ struct vr_interface {
 #elif defined(__FreeBSD__)
     struct mbuf;
     void (*saved_if_input) (struct ifnet *, struct mbuf *);
-#endif
-#elif defined(_NTKERNEL)
+#elif defined(_WIN64)
```
Comment (jablonskim) [inline include/vr_interface.h]: Please change it to `_WIN32`

File: include/vr_os.h [hunk 4]
```diff
@@ -93,9 +93,9 @@ typedef unsigned int __u32;
 #define ASSERT(x) assert((x));
 #endif
 #endif /* __FreeBSD__ */
-#if defined(_WINDOWS)
+#if defined(_WIN32)
 #include "windows_builtins.h"
-#ifdef _NTKERNEL
+#ifdef _KERNEL_MODE
```
Comment (jablonskim) [inline include/vr_os.h]: Maybe change it to `__KERNEL__`

File: include/vr_os.h [hunk 5]
```diff
@@ -108,7 +108,7 @@ typedef unsigned int __u32;
 #pragma warning(disable : 4701)     // potentially uninitialized local variable used
 #pragma warning(disable : 4706)     // assignment within conditional expression
 
-#endif /* _NTKERNEL */
+#endif /* _KERNEL_MODE */
```
Comment (jablonskim) [inline include/vr_os.h]: Maybe change it to `__KERNEL__`

File: include/vr_os.h [hunk 6]
```diff
@@ -108,7 +108,7 @@ typedef unsigned int __u32;
...
 #endif /* _WINDOWS */
```
Comment (jablonskim) [inline include/vr_os.h]: Please change it to `_WIN32`

File: include/vr_common.h [hunk 7]
```diff
@@ -21,19 +21,21 @@ typedef BOOLEAN bool;
...
 #include <stdint.h>
 #include <stdbool.h>
 
-#endif /* _NTKERNEL */
-
 #define __LITTLE_ENDIAN 1
 #define __BIG_ENDIAN 2
 #define __BYTE_ORDER __LITTLE_ENDIAN
 
+#endif
```
Comment (katrybacka) [inline include/vr_common.h]: Maybe #endif /* __KERNEL__ */ would be more clear

**Pull request merged**